priv/cabbage/apps/itest/test/itest/payment_v2_test.exs

```diff
@@ -50,7 +51,10 @@ defmodule PaymentV2Test do
     balance = Client.get_exact_balance(alice.account, expecting_amount)
     balance = balance["amount"]
 
-    assert_equal(expecting_amount, balance, "For #{alice.account}")
+    case expecting_amount do
+      0 -> assert(balance == nil, "Alice ETH amount is not 0")
```

Comment:
`balance == nil` is not obvious and misleading.
Can we set it to zero, kind of (line above): `balance = Map.get(balance, "amount", 0)`

Reply:
lol, TIL in elixir you can try to access a value from nil and would return nil instead of crashing.
the `Client.get_exact_balance` would actually return `nil` instead of empty map. 

I change to code to the following instead:
```
Client.get_exact_balance(bob.account, expecting_amount)["amount"] || 0
```